modules/webhook-configuring-acs.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-using-generic-webhooks.adoc
+:_module-type: PROCEDURE
+[id="webhook-configuring-acs_{context}"]
+= Configuring {product-title}
+
+[role="_abstract"]
+Create a new integration in {product-title} by using the webhook URL.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Notifier Integrations* section and select *PagerDuty*.
+. Click *New Integration* (add icon).
+. Enter a name for *Integration Name*.
+. Enter the webhook URL in the *Endpoint* field.
+. If your webhook receiver uses an untrusted certificate, enter a CA certificate in the *CA Cert* field, otherwise leave it blank.
++
+[NOTE]
+====
+The server certificate used by the webhook receiver must be valid for the endpoint DNS name.
+You can turn on the *Skip TLS Verify* toggle to ignore this validation.
+Red Hat does not recommend turning off TLS verification.
+====
+. Optional: Turn on the *Enable Audit Logging* toggle, to receive alerts about all the changes made in {product-title}.
+//See link:/docs/integrate-with-other-tools/enable-audit-logging/[Audit Logging] for more information.
++
+[NOTE]
+====
+Red Hat recommends using separate webhooks for alerts and audit logs to handle these messages differently.
+====
+. To authenticate with the webhook receiver, enter details for:
+** *Username* and *Password* for basic HTTP authentication, or
+** Custom *Header*, for example: `Authorization: Bearer <access_token>`.
```

Comment:
I missed this before but I recommend restructuring the lead-in sentence and list to something like this:
```
. To authenticate with the webhook receiver, enter details for one of the following:
** *Username* and *Password* for basic HTTP authentication
** Custom *Header*, for example: `Authorization: Bearer <access_token>`
```

This applies a few ISG principles:
- "Generally, use a sentence to introduce a list that is not a procedure." ("details for" --> "details for one of the following")
- "do not use a conjunction at the beginning or end of a list item" ("or" in your first bullet)
- "do not include any end punctuation if list items comprise only
sentence fragments" (so no period needed at the end of second bullet)